lib/experimental/fonts-api/fonts-api.php

```diff
@@ -197,6 +198,11 @@ function wp_print_fonts( $handles = false ) {
 
 		_wp_scripts_maybe_doing_it_wrong( __FUNCTION__ );
 
+		// Print all registered fonts for the iframed editor.
+		if ( true === $handles ) {
+			$handles = $registered;
+		}
+
```

Comment:
Shouldn't this condition be moved _above_ the `if ( empty( $handles ) ) {` check a few lines above? 🤔